quickfixj-messages/quickfixj-messages-all/pom.xml

```diff
@@ -59,140 +52,76 @@
-						</configuration>
-					</execution>
-					<execution>
-						<id>fix41</id>
-						<goals>
-							<goal>generate</goal>
-						</goals>
-						<configuration>
-							<dictFile>../quickfixj-messages-fix41/src/main/resources/FIX41.xml</dictFile>
-							<packaging>quickfix.fix41</packaging>
-							<fieldPackage>quickfix.field</fieldPackage>
-							<decimal>${generator.decimal}</decimal>
-						</configuration>
-					</execution>
-					<execution>
-						<id>fix40</id>
-						<goals>
-							<goal>generate</goal>
-						</goals>
+						<phase>prepare-package</phase>
+						<goals><goal>add-source</goal></goals>
 						<configuration>
-							<dictFile>../quickfixj-messages-fix40/src/main/resources/FIX40.xml</dictFile>
-							<packaging>quickfix.fix40</packaging>
-							<fieldPackage>quickfix.field</fieldPackage>
-							<decimal>${generator.decimal}</decimal>
-						</configuration>
-					</execution>
-					<execution>
-						<id>fix50sp2</id>
-						<goals>
-							<goal>generate</goal>
-						</goals>
-						<configuration>
-							<dictFile>../quickfixj-messages-fix50sp2/src/main/resources/FIX50SP2.modified.xml</dictFile>
-							<packaging>quickfix.fix50sp2</packaging>
-							<fieldPackage>quickfix.field</fieldPackage>
-							<decimal>${generator.decimal}</decimal>
+							<sources>
+								<source>${project.basedir}/../../quickfixj-core/target/generated-sources/</source>
+							</sources>
 						</configuration>
 					</execution>
+				</executions>
+			</plugin>
+			
+			<plugin>   
+				<groupId>org.apache.felix</groupId>
+				<artifactId>maven-bundle-plugin</artifactId>
+				<executions>
 					<execution>
-						<id>fix50sp1</id>
-						<goals>
-							<goal>generate</goal>
-						</goals>
+						<id>bundle-manifest</id>
+						<phase>process-classes</phase>
+						<goals>    
+							<goal>manifest</goal>
+						</goals>   
 						<configuration>
-							<dictFile>../quickfixj-messages-fix50sp1/src/main/resources/FIX50SP1.modified.xml</dictFile>
-							<packaging>quickfix.fix50sp1</packaging>
-							<fieldPackage>quickfix.field</fieldPackage>
-							<decimal>${generator.decimal}</decimal>
+							<instructions>
+								<Export-Package>quickfix.field;version="${project.version}";uses:=quickfix,quickfix.fix41;version="${project.version}";uses:=quickfix,quickfix.field;version="${project.version}"</Export-Package>
+							    <Import-Package>quickfix,quickfix.field;version="${project.version}"</Import-Package>
+								<Require-Capability>osgi.ee;filter:="(&#38;(osgi.ee=JavaSE)(version=${jdkLevel}))"</Require-Capability>								
+							</instructions>
 						</configuration>
 					</execution>
 				</executions>
 			</plugin>
 			
 			<plugin>
-				<artifactId>maven-surefire-plugin</artifactId>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-jar-plugin</artifactId>
+				<version>${maven-jar-plugin-version}</version>
 				<configuration>
-					<argLine>-Xmx512m -XX:MaxPermSize=128m</argLine>
```

Comment:
@chrjohn no, it is at that line but the plugin should be surefire-plugin not jar-plugin.